contrib/for-tests/mount-tester/mt.go

```diff
@@ -143,3 +151,30 @@ func readWriteNewFile(path string) error {
 
 	return readFileContent(path)
 }
+
+func readFileContentInLoop(path string) error {
+	if path == "" {
+		return nil
+	}
+	var (
+		contentBytes []byte
+		err          error
+	)
+	start := time.Now()
+	for time.Now().Sub(start) < (500 * time.Second) {
```

Comment:
Please use our standard wait.Poll() utility function here, rather than this loop.